docs/ZoneDocumentation.md

```diff
@@ -0,0 +1,41 @@
+# Zone Documentation
+
+The `zone` of the order is an optional secondary account attached to the order with two additional privileges:
+
+1. The zone may cancel orders where it is named as the zone by calling `cancel`. (Note that offerers can also cancel their own orders, either individually or for all orders signed with their current counter at once by calling `incrementCounter`).
+2. "Restricted" orders (as specified by the order type) must either be executed by the zone or the offerer, or must be approved as indicated by a call to an `isValidOrder` or `isValidOrderIncludingExtraData` view function on the zone.
+
+An example zone contract implementation can be found at `/contracts/zones/GlobalPausable.sol`
+
+The `GlobalPausable` contract can be used by its deployer to cancel specific orders, execute fulfillment on restricted order, and pause all orders which use it as a zone.
+
+## Further implementation
+
+We could use something like this, to allow the owner of the zone to execute restricted orders, but it's not entirely necessary:
```

Comment:
Rephrase throughout to remove the use of “we”

Reply:
this has been implemented so theres no reason to add a "could be implemented"